src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/adyen/partialPayment.js

```diff
@@ -0,0 +1,44 @@
+const Logger = require('dw/system/Logger');
+const Transaction = require('dw/system/Transaction');
+const Money = require('dw/value/Money');
+const AdyenConfigs = require('*/cartridge/scripts/util/adyenConfigs');
+const adyenCheckout = require('*/cartridge/scripts/adyenCheckout');
+
+function makePartialPayment(req, res, next) {
+  try {
+    const request = JSON.parse(req.body);
+
+    const { paymentMethod, splitPaymentsOrder, amount } = request;
+
+    const partialPaymentRequest = {
+      merchantAccount: AdyenConfigs.getAdyenMerchantAccount(),
+      amount,
+      reference: 'partialPaymentRef',
+      paymentMethod,
+      order: splitPaymentsOrder,
+    };
+
+    const response = adyenCheckout.doPaymentsCall(0, 0, partialPaymentRequest);
```

Comment:
I understand why these are `0`, but only after checking input args for the doPaymentsCall function. Maybe we can add some comments and/or input `null` or `undefined` or something